src/misc.js

```diff
@@ -33,7 +33,7 @@ function enforceRangeLong( value ) {
 // because to load it in a <img>,
 // the type must be set to this value
 function blobCanBeSVG( blob ) {
-  return blob?.type === "image/svg+xml";
+  return blob && blob.type === "image/svg+xml";
```

Comment:
I'm not sure about this change though. We are already using rest params et al. and async/await all over the place, I'm not convinced the compatibility would be greatly improved by this change only.

Reply:
Well, this change is a fix for a syntax error when testing on iPhone 6 (iOS12): 
```
Unexpected token '.'
```
Just make this change and it works fine.

Reply:
Oh, I didn't know Safari had such a gap between this feature and e.g async/await.
Since it's the only occurrence I'll merge it like that. Thanks again.